.github/workflows/config.yml

```diff
@@ -78,6 +74,24 @@ jobs:
           path: coverage
           name: coverage
 
+  measure-tests:
+    runs-on: ubuntu-latest
+    needs: [format-files]
+    container:
+      image: docker://nrel/openstudio:3.8.0
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Install gems
+        run: |
+          rm -f Gemfile.lock && bundle install
+
+      - name: Run all measure tests
+        run: |
+          bundle exec rake workflow:measure_tests
```

Comment:
Move measure_tests into its own namespace and job.

Reply:
@joseph-robertson : This new test increases the CI time by about 27 minutes. I take it that is just because we used 10k samples? Is it worth reducing the sample size slightly?

Reply:
We could. Perhaps pick a random 5K subset from the already-sampled 10K set? Something like that?

Reply:
@afontani I tried running 8K out of the 10K (random seeded 80%), and the resulting CI runtime seems to not add much extra, if any, overall runtime. Let me know what you think.